client/app/screens/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Login from 'react-native-login';
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
So I'm using the class from font awesome because it gives use the very nice/easy icons to work with for whichever social providers we try to use.  It is a justifiable deviation.